[2024-10-22T03:18:22.790+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-22T03:18:22.803+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: auto_training_crawling_data_okx.process_data_task manual__2024-10-22T03:18:08.398519+00:00 [queued]>
[2024-10-22T03:18:22.810+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: auto_training_crawling_data_okx.process_data_task manual__2024-10-22T03:18:08.398519+00:00 [queued]>
[2024-10-22T03:18:22.810+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-22T03:18:22.831+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): process_data_task> on 2024-10-22 03:18:08.398519+00:00
[2024-10-22T03:18:22.845+0000] {standard_task_runner.py:72} INFO - Started process 377152 to run task
[2024-10-22T03:18:22.855+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'auto_training_crawling_data_okx', 'process_data_task', 'manual__2024-10-22T03:18:08.398519+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/full_pipeline.py', '--cfg-path', '/tmp/tmpj9drsq4c']
[2024-10-22T03:18:22.857+0000] {standard_task_runner.py:105} INFO - Job 42: Subtask process_data_task
[2024-10-22T03:18:22.910+0000] {task_command.py:467} INFO - Running <TaskInstance: auto_training_crawling_data_okx.process_data_task manual__2024-10-22T03:18:08.398519+00:00 [running]> on host ecs-gpu
[2024-10-22T03:18:23.325+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='auto_training_crawling_data_okx' AIRFLOW_CTX_TASK_ID='process_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-22T03:18:08.398519+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-22T03:18:08.398519+00:00'
[2024-10-22T03:18:23.327+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-22T03:18:23.603+0000] {_client.py:1038} INFO - HTTP Request: GET https://www.okx.com/api/v5/market/candles?instId=BTC-USDT&bar=4H&limit=300 "HTTP/2 200 OK"
[2024-10-22T03:18:23.620+0000] {logging_mixin.py:190} INFO - Shape của DataFrame: (180, 9)
[2024-10-22T03:18:23.622+0000] {logging_mixin.py:190} INFO - Thời gian bắt đầu: 2024-09-22 04:00:00
[2024-10-22T03:18:23.622+0000] {logging_mixin.py:190} INFO - Thời gian kết thúc: 2024-10-22 00:00:00
[2024-10-22T03:18:23.911+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/airflow/dags/full_pipeline.py", line 152, in process_data
    df = scaler.fit_transform(crawled_data)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_set_output.py", line 157, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 916, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 435, in fit
    return self.partial_fit(X, y)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 1152, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 473, in partial_fit
    X = self._validate_data(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 605, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/validation.py", line 915, in check_array
    array = _asarray_with_order(array, order=order, dtype=dtype, xp=xp)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_array_api.py", line 380, in _asarray_with_order
    array = numpy.asarray(array, order=order, dtype=dtype)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/pandas/core/generic.py", line 1998, in __array__
    arr = np.asarray(values, dtype=dtype)
TypeError: float() argument must be a string or a number, not 'Timestamp'
[2024-10-22T03:18:23.924+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=auto_training_crawling_data_okx, task_id=process_data_task, run_id=manual__2024-10-22T03:18:08.398519+00:00, execution_date=20241022T031808, start_date=20241022T031822, end_date=20241022T031823
[2024-10-22T03:18:23.950+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-22T03:18:23.951+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 42 for task process_data_task (float() argument must be a string or a number, not 'Timestamp'; 377152)
Traceback (most recent call last):
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/airflow/dags/full_pipeline.py", line 152, in process_data
    df = scaler.fit_transform(crawled_data)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_set_output.py", line 157, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 916, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 435, in fit
    return self.partial_fit(X, y)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 1152, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 473, in partial_fit
    X = self._validate_data(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 605, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/validation.py", line 915, in check_array
    array = _asarray_with_order(array, order=order, dtype=dtype, xp=xp)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_array_api.py", line 380, in _asarray_with_order
    array = numpy.asarray(array, order=order, dtype=dtype)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/pandas/core/generic.py", line 1998, in __array__
    arr = np.asarray(values, dtype=dtype)
TypeError: float() argument must be a string or a number, not 'Timestamp'
[2024-10-22T03:18:23.987+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-22T03:18:24.005+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-22T03:18:24.006+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
